fe/src/App.vue

```diff
@@ -15,4 +15,6 @@ export default {
     NavBar,
   },
 };
+
```

Comment:
is this space necessary?

Reply:
I removed the extra space already po ma'am.